docs/_content/standaard/standaard.md

```diff
@@ -529,6 +529,24 @@ op `true` zetten.
 Indien de laatste gebruiksrechten op een informatieobject verwijderd worden,
 dan MOET de indicatie weer op `null` gezet worden.
 
+#### Archiveren
+
+**Vernietigen van informatieobjecten**
+
+Bij `DELETE` requests op informatieobjecten MOETEN het informatieobject en 
+gerelateerde objecten fysiek uit de opslag verwijderd worden indien er geen 
+relaties meer zijn met andere objecten via objectinformatieobject.
+
+Indien er nog relaties zijn moet een 400 worden teruggegeven.
```

Comment:
Consistente formulering
```suggestion
Indien er nog relaties zijn, dan MOET het DRC antwoorden met een `HTTP 400` foutbericht.
```

Reply:
Op de huidige referentie-implementatie is er een cascade-delete op `ObjectInformatieObject`. Dit moet dus een `models.PROTECT` worden (revert van de change?) die vertaalt naar een `ValidationError`, niet?

Reply:
Ik bedacht dit bij het schrijven. Het is tevens de vraag wat jij er van vind.

Reply:
Ik weet het niet, cascade delete is erg convenient & developer friendly, maar ook wel een serieus risico om onbedoeld relaties te verbreken, zeker als een IO ook in niet-zaakgericht werken gebruikt wordt. Daarom neig ik naar een `models.PROTECT`, dus de relaties moeten expliciet verwijderd worden.

Reply:
Dit hangt samen met: https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/pull/43#pullrequestreview-226748783

Architectuur wise even over bomen.

Reply:
Ik stop dit stukje er in en laat bovenstaand PR als leading issue voor wat nu deze PR blokt.